src/components/base/ContentContainer/index.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import { BORDER_STYLE } from '@utils/constants';
+import { BORDER_STYLE, FILTER_LIST } from '@utils/constants';
```

Comment:
```suggestion
import { BORDER_STYLE } from '@utils/constants';
```
이거 안썼다고 콘솔에 경고뜨는데 삭제만 좀 부탁드립니다!

Reply:
삭제했습니다.